marklogic-client-api/src/main/java/com/marklogic/client/datamovement/JacksonCSVSplitter.java

```diff
@@ -44,20 +44,36 @@
     private long count = 0;
     private ArrayNode headers = null;
     
+    /**
```

Comment:
My understanding of standard JavaDoc practice is to provide the sentence describing the method as well as the description of the return value.

Or, when you generate the JavaDoc from the sources, does it look okay?

The main concern is that we want it to be intelligible for readers.

The same question applies to other getters here and in Splitter that have a return description but not a method description.